### ISSUE TITLE
Outdated xacros in dependencies/ur_description

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/robot_movement_interface#2]
The urdfs / xacros in the `dependencies/ur_description` package are rather outdated: they do not include the `base` or `tool0` frames, nor does the package include the recently added UR3 model.

Would it be an option to just add the `ur_description` from the `universal_robot` repository to the `ur_driver` package (in this repository) `run_depend`s instead of duplicating it?

**Comment by gavanderhoorn** [comment]
The `ur_description` package in `dependencies` also collides with the package in `ros-industrial/universal_robot`, making a workspace with both impossible to build.

**Comment by pablo-quilez** [comment]
Yes, sure, adding the dependency should be fine and the right way to do it.

2015-11-30 18:37 GMT+01:00 G.A. vd. Hoorn <notifications@github.com>:
